build_raspOVOS.sh

```
@@ -29,118 +91,89 @@ fi
+sudo chmod +x /usr/libexec/ovos*
+sudo chmod +x /usr/local/bin/ovos*
+
+echo "Setting up libcamera..."
+apt install -y python3-libcamera python3-kms++ libcap-dev
+
+echo "Installing uv and sdnotify..."
+pip install sdnotify uv --break-system-packages
+
+echo "Installing Pipewire..."
+apt install -y pipewire pipewire-alsa alsa-utils
+SOUND_SERVER="pipewire"
```

Shellcheck (0.10.0)

[warning] SOUND_SERVER appears unused. Verify use (or export if used externally).

(SC2034)